examples/springdoc-openapi-scala-2/simple/README.md

```diff
@@ -0,0 +1,41 @@
+# springdoc-openapi-scala-2 Example Application
+
+This is a simple example application demonstrating the use of springdoc-openapi-scala with Spring Boot 3.x.
+
+## Prerequisites
+
+- JDK 17 or higher
+- SBT 1.x
+- Scala 2.12.x
+
+## Running the Application
+
+To run the application with hot-reload enabled:
+
+```bash
+cd springdoc-openapi-scala/examples/springdoc-openapi-scala-2/simple
```

Comment:
```suggestion
cd springdoc-openapi-scala/examples/springdoc-openapi-scala-2/simple
```suggestion
cd springdoc-openapi-scala/examples/springdoc-openapi-scala-1/simple
sed -i '' 's/`springdoc-openapi-scala-2-version`: String = \?\?\?/`springdoc-openapi-scala-2-version`: String = "0.3.2"/g' build.sbt # Omit '' after the -i flag if not on mac
```
```